src/KeyVault/KeyVault/Commands/RemoveAzureKeyVault.cs

```diff
@@ -54,7 +54,8 @@ public class RemoveAzureKeyVault : KeyVaultManagementCmdletBase
             HelpMessage = "Specifies the name of the key vault to remove.")]
         [ResourceNameCompleter("Microsoft.KeyVault/vaults", "ResourceGroupName")]
         [ValidateNotNullOrEmpty]
-        public string VaultName { get; set; }
+        [Alias("VaultName")]
+        public string Name { get; set; }
```

Comment:
Is the original reference to VaultName required to be modified?

Reply:
So @Skuldo , is there any particular reason why not keep `VaultName` as-is, and add `Name` as an alias?